experimental/include/experimental/Transforms/LSQSizing/LSQSizingSupport.h

```diff
@@ -0,0 +1,164 @@
+#include "dynamatic/Support/TimingModels.h"
+#include "dynamatic/Transforms/BufferPlacement/CFDFC.h"
+#include <list>
+#include <mlir/IR/Operation.h>
+#include <optional>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#ifndef DYNAMATIC_TRANSFORMS_LSQSIZING_LSQSIZINGSUPPORT_H
+#define DYNAMATIC_TRANSFORMS_LSQSIZING_LSQSIZINGSUPPORT_H
+
+namespace dynamatic {
+namespace experimental {
+namespace lsqsizing {
+
+// Define a structure for a graph node
```

Comment:
By convention all entity comments (e.g., struct, class, functions, members, etc.) should start with `///` instead of `//`. This is so that automated documentation parsing tools can recognize them more easily (I think). Please try to make a quick pass over your files and use `///` in those scenarios.